skbio/io/format/embl.py

```diff
@@ -0,0 +1,1568 @@
+    # FT                   to be right,based on 454- and PacBio-sequencing)"
+    # I could express features by resizing sequence, however the resulting
+    # sequence will be different than data read. I decide to mantain
+    # sequences indentical to read values, and to discard features
+    # for such entries
+
+    # result array
+    sequence = []
+
+    for line in lines:
+        # ignore record like:
+        # SQ   Sequence 275 BP; 64 A; 73 C; 88 G; 50 T; 0 other;
+        if line.startswith('SQ'):
+            continue
+
+        # remove the numbers inside strings. revome spaces around string
+        line = ''.join([i for i in line.strip() if not i.isdigit()])
+
+        # remove space from sequence
+        line = line.replace(" ", "")
+
+        # append each line sequence to sequence list
+        sequence.append(line)
+
+    return ''.join(sequence)
+
+
+def _serialize_sequence(obj, indent=5):
+    '''Serialize seq to SQ.
+
+    Parameters
+    ----------
+    obj : DNA, RNA, Sequence Obj
+    '''
+
+    # magic numbers
+    n = 1
+    line_size = 60
+    frag_size = 10
+
+    # count bases in sequence
+    seq = str(obj).lower()
+    n_a = seq.count("a")
+    n_c = seq.count("c")
+    n_g = seq.count("g")
+    n_t = seq.count("t")
+    n_others = len(obj) - (n_a + n_c + n_g + n_t)
+
+    # define SQ like this:
+    # SQ   Sequence 275 BP; 63 A; 72 C; 88 G; 52 T; 0 other;
+    SQ = "SQ   Sequence {size} {unit}; {n_a} A; {n_c} C; {n_g} G; " +\
+         "{n_t} T; {n_others} other;\n"
+
+    # apply format
+    SQ = SQ.format(size=len(obj), unit=obj.metadata["LOCUS"]["unit"].upper(),
+                   n_a=n_a, n_c=n_c, n_g=n_g, n_t=n_t, n_others=n_others)
+
+    for i in range(0, len(seq), line_size):
+        line = seq[i:i+line_size]
+        s = '{key:<{indent}}{s}'.format(
+            key='', indent=indent, s=chunk_str(line, frag_size, ' '))
+        # pad string left and right
+        s = '{:<70}'.format(s) + '{:>10}\n'.format(i+len(line))
+        if n == 1:
+            # Add SQ header to sequence
+            s = SQ + s
+        n = n + line_size
+        yield s
+
+
+def _embl_parse_feature_table(lines, metadata):
+    """Parse embl feature tables"""
+
+    # get size of the feature
+    length = metadata["LOCUS"]["size"]
+
+    if "PARENT_ACCESSION" in metadata:
+        # this is a feature-level-products entry and features are relative
+        # to parent accession; in the same way a subset of a Sequence objcet
+        # has no interval metadata, I will refuse to process interval
+        # metadata here
+        return
+
+    # define interval metadata
+    imd = IntervalMetadata(length)
+
+    # remove feature header table
+    idxs = [i for i, line in enumerate(lines) if line.startswith("FH")]
+    lines = [line for i, line in enumerate(lines) if i not in idxs]
+
+    # remove FT from the header
+    lines = [line.replace("FT", "  ", 1) for line in lines]
+
```

Comment:
Ok. And magik number shuld be 19, since I removed (with no replacement) two chars from the string